src/dynd/types/categorical_type.cpp

```diff
@@ -469,19 +469,6 @@ struct get_ints_kernel : nd::base_kernel<get_ints_kernel> {
   }
 };
 
-static const std::map<std::string, nd::callable> &categorical_array_properties()
```

Comment:
Apparently there are no tests for this feature. Should this be handled in array.cpp like for structs?